etc/picongpu/bash-devServer-hzdr/mpiexec.tpl

```diff
@@ -18,6 +18,36 @@
 # If not, see <http://www.gnu.org/licenses/>.
 #
 
+help()
+{
+  echo "PIConGPU submit script generated with tbg"
+  echo ""
+  echo "usage: $0 [--verify]"
+  echo ""
+  echo "--validate      - validate picongpu call instead of running the simulation"
+  echo "--h | --help    - print this help message"
+}
+
+VALIDATE_MODE=false
+for arg in "$@"; do
+  case $arg in
+  --validate)
+    VALIDATE_MODE=true
+    shift # Remove --skip-verification from `$@`
```

Comment:
Outdated naming of the flag in the comment (`--skip-verification`)?

Reply:
ah yes, copy paste mistakes...